lib/react_client/react_interop.dart

```diff
@@ -49,7 +49,7 @@ abstract class React {
   external static JsRef createRef();
   external static ReactClass forwardRef(Function(JsMap props, JsRef ref) wrapperFunction);
   external static ReactClass memo(
-    dynamic Function(JsMap props, [JsMap legacyContext]) wrapperFunction, [
+    dynamic wrapperFunction, [
```

Comment:
I had to loosen this to be able to pass in JS `ReactClass`es; this should not be considered a breaking change